nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -440,6 +440,22 @@ The module update takes care of the new config syntax and the data itself (user
 
 - `fusuma` now enables the following plugins: [appmatcher](https://github.com/iberianpig/fusuma-plugin-appmatcher), [keypress](https://github.com/iberianpig/fusuma-plugin-keypress), [sendkey](https://github.com/iberianpig/fusuma-plugin-sendkey), [tap](https://github.com/iberianpig/fusuma-plugin-tap) and [wmctrl](https://github.com/iberianpig/fusuma-plugin-wmctrl).
 
+- `blender` now provides both the lastest stable and LTS versions under `blenderVersions`. `blender` and `blender-hip` now default to `blenderVersions.stable`, which points to the latest stable version. The latest LTS version is available with `blender-lts`. Explicitly versioned packages are available with `blenderVersions.blender_3_3` and `blenderVersions.blender_3_6`, as well as their `blender-hip_3_x` variants. An attribute `isLTS` is added.
```

Comment:
We should not advertise explizit minor versions as they are going to be removed without aliases.